store/conf/attrs/zimbra-attrs.xml

```diff
@@ -10474,7 +10474,7 @@ TODO: delete them permanently from here
   <desc>Define body: field for 2FA email in html format</desc>
 </attr>
 
-<attr id="4116" name="zimbraHideAliasesInGal" type="boolean" cardinality="single" optionalIn="account" flags="accountInfo" since="10.1.0">
+<attr id="4116" name="zimbraHideAliasesInGal" type="boolean" cardinality="single" optionalIn="account,cos,domain" flags="accountInfo,accountCosDomainInherited" since="10.1.0">
```

Comment:
@desouzas please comment if attribute name is required here. This attribute was earlier created keeping current requirement in mind, there are no other references to it.